ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/TeamStatusPanel.java

```diff
@@ -66,59 +65,41 @@ public void updateStatus(int activeTeamId, Status status) {
     }
 
     private void initComponents() {
-        GridBagConstraints constraints = new GridBagConstraints();
+        setLayout(new MigLayout("", "[100:pref,fill]", "[100:pref,fill]"));
 
-        setLayout(new GridBagLayout());
+        mJailPanel.addComponentListener(new ComponentListener() {
```

Comment:
This is adding a `ComponentListener` to `mJailPanel` which calls another method on `mJailPanel`....can the listener just go directly IN `JailPanel`?